src/components/ui/menu-option/menu-option.scss

```diff
@@ -5,22 +5,16 @@
 /* the height of each item/row */
 $menu-item-height: 40px;
 
-/* padding at the top and bottom of each group of items */
-$menu-group-padding: 4px;
-
-/* the padding of each item (either side of the text ) */
-$menu-item-padding: 12px;
-
 .kui-theme--light {
-  --menu-option-border-width: 1px 0 0 2px;
   --menu-option-border-color: #{rgba(black, 0.06)};
-  --menu-option-bg-hover: #{rgba(white, 0.3)};
+  --menu-option-bg-hover: #{$color-bg-light-2};
```

Comment:
This is where I mean to to put this comment:

Looking at the design, I think this one is supposed to be `color-bg-dark-3`.

Reply:
Yes, you're right. Will change it, thanks